docs/GasZipFacet.md

```diff
@@ -0,0 +1,37 @@
+# GasZipFacet
+
+## Description
+
+The GasZipFacet provides function to deposit ERC20 and native tokens to the Gas.zip protocol (https://www.gas.zip/)
+
+## How To Use
```

Comment:
Please add the new bridge functions to the docs

Reply:
updated. Thanks for pointing it out